app/src/main/res/layout/activity_main.xml

```diff
@@ -218,15 +220,15 @@
                 android:layout_marginHorizontal="10dp"
                 android:layout_weight="2.6"
                 android:background="@drawable/gray_2_circle"
-                android:text="0                   "
+                android:text="0                  "
                 android:textSize="22sp" />
 
             <Button
                 android:id="@+id/dot"
                 android:layout_width="60dp"
                 android:layout_height="70dp"
                 android:layout_marginHorizontal="10dp"
-                android:layout_weight=".47"
+                android:layout_weight=".472"
                 android:background="@drawable/gray_2_circle"
                 android:text="@string/dot"
                 android:textSize="22sp" />
```

Comment:
should be able to just add the size using `type.xml` and updating your `theme.xml` to point to your custom button type